pymaker/oracles.py

```diff
@@ -46,7 +47,11 @@ def __init__(self, web3: Web3, address: Address):
     def poke(self) -> Transact:
         return Transact(self, self.web3, self.abi, self.address, self._contract, 'poke', [])
 
-    def __repr__(self):
-        return f"OSM('{self.address}')"
+    def peek(self) -> Wad:
+        return Wad(Web3.toInt(self.web3.eth.getStorageAt(self.address.address, 3)[16:]))
```

Comment:
should be returning always a Wad? (that is even in case we add an collateral with different precision?)

Reply:
The contracts don't expose the precision.  Until they do, it's reasonable to treat all collateral types with the same precision, as the `join` contracts do.

Reply:
:+1: